src/pudl/extract/__init__.py

```diff
@@ -17,6 +17,7 @@
     eia861,
     eia923,
     eia930,
+    eia_aeo,
```

Comment:
Existing data source codes don't use an underscore to separate the "agency" and the "form", including the EIA AEO archive / datastore entry, so I think we probably should stick with `eiaaeo` unless we want to update them all to use underscores. `eia_bulk_elec` is the only outlier.